src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
+type Props = {
+  todos: Todo[]
+  handleDataModal: (todo: Todo) => void
+  setOpenModal: (openModal: boolean) => void
+  selectedTodo: Todo | null
+};
+
+export const TodoList: React.FC<Props> = (
+  {
+    todos, handleDataModal, setOpenModal, selectedTodo,
+  },
+) => (
```

Comment:
```suggestion
export const TodoList: React.FC<Props> = ({
  todos,
  handleDataModal,
  setOpenModal,
  selectedTodo,
}) => (
```
Improve your code style.